tests/js/spec/views/organizationIntegrations/integrationRow.spec.jsx

```diff
@@ -0,0 +1,218 @@
+    it('is an internal SentryApp', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="sentryApp"
+          slug="my-headband-washer-289499"
+          displayName="My Headband Washer"
+          status="Installed"
+          publishStatus="internal"
+          configurations={0}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual(
+        'my-headband-washer-289499'
+      );
+      expect(wrapper.find('IntegrationName').props().children).toEqual(
+        'My Headband Washer'
+      );
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/sentry-apps/my-headband-washer-289499/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').props().status).toEqual('internal');
+    });
+
+    it('is a published SentryApp', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="sentryApp"
+          slug="clickup"
+          displayName="ClickUp"
+          status="Not Installed"
+          publishStatus="published"
+          configurations={0}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('clickup');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('ClickUp');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/sentry-apps/clickup/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Not Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+    });
+  });
+  describe('First Party Integration', function() {
+    it('has been installed (1 configuration)', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="bitbucket"
+          displayName="Bitbucket"
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('bitbucket');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Bitbucket');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/integrations/bitbucket/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').props().children).toEqual('1 Configuration');
+      expect(wrapper).toMatchSnapshot();
```

Comment:
Why snapshot this and not the SentryApp tests?  The snapshots seems a bit redundant given the above expectations.